app/src/app/initApp.js

```diff
@@ -54,14 +54,21 @@ export default () => {
     height: 1000
   })
 
+  // figure out if we start in background
+  const autostart = (process.argv || []).indexOf('--autostart') !== -1
+  const startInBackground = autostart && env.startInBackgroundWhenAutostarted
+  console.log(`autostart: ${autostart}`)
```

Comment:
do we need the logs in production?

Reply:
yes, it's quite useful for debugging issues with configuration at customer's computers. I tried to be careful to not log too much

Reply:
We can't just log everything everywhere, this will result in a huge amount of logs and make it useless all together.  On the client I use "debug" package which allows to turn on specific log types as needed. We should be able to use that here as well.

Reply:
it's just one line at the beginning of the app start, helps a lot with debugging problems